src/main/java/de/hysky/skyblocker/skyblock/entity/MobBoundingBoxes.java

```diff
@@ -30,14 +33,25 @@ public static boolean shouldDrawMobBoundingBox(Entity entity) {
 			String name = entity.getName().getString();
 
 			return switch (entity) {
-				case PlayerEntity _p when name.equals("Lost Adventurer") || name.equals("Shadow Assassin") || name.equals("Diamond Guy") -> SkyblockerConfigManager.get().dungeons.starredMobBoundingBoxes;
+				case PlayerEntity _p when name.equals("Lost Adventurer") || name.equals("Shadow Assassin") || name.equals("Diamond Guy") ->
+						SkyblockerConfigManager.get().dungeons.starredMobBoundingBoxes;
 				case ArmorStandEntity _armorStand -> false;
 
 				// Regular Mobs
 				default -> SkyblockerConfigManager.get().dungeons.starredMobBoundingBoxes && MobGlow.isStarred(entity);
 			};
 		}
 
+		if (SkyblockerConfigManager.get().slayers.highlightMinis.toString().equals("HITBOX") && entity instanceof ArmorStandEntity le && SlayerMobs.isSlayerMiniMob(le)) {
```

Comment:
You know you can do `SkyblockerConfigManager.get().slayers.highlightMinis == HighlightSlayerEntities.HITBOX` right? Same for below and also `MobGlow`.